### PR TITLE
Knative Eventing Mesh Migration Guide

### PR DESCRIPTION
**Pull request by montaro**
See also  #7236

### CODE REVIEW

File: docs/migration-guides/1.10-1.11.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+1. List all the applications:
+
+    ```bash
+     kubectl get applications 
+    ```  
+2. Check if each Application has a corresponding HTTP Source Adapter. 
+
+    ```bash
+    kubectl -n kyma-integration get pods
```
Comment (k15r) [inline docs/migration-guides/1.10-1.11.md]:
```suggestion
    kubectl -n kyma-integration get httpsource
```
imho checking for an httpsource makes more sense here

File: docs/migration-guides/1.10-1.11.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    ```bash
+     kubectl get applications 
+    ```  
+2. Check if each Application has a corresponding HTTP Source Adapter. 
+
+    ```bash
+    kubectl -n kyma-integration get pods 
+    ```
+3. Check if the Channel status is `READY`
```
Comment (k15r) [inline docs/migration-guides/1.10-1.11.md]:
```bash
kubectl -n kyma-integration get channel
```

File: docs/migration-guides/1.10-1.11.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+4. Check if Kyma Subscriptions have been converted to Knative Triggers:
+
+    ```bash
+    kubectl -n test get triggers
```
Comment (k15r) [inline docs/migration-guides/1.10-1.11.md]:
```suggestion
    kubectl --all-namespaces get triggers 
```

would it make sense to check for remaining subscriptions as well?   
```bash
k get -A subscriptions
```

File: docs/migration-guides/1.10-1.11.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+    ```bash
+    kubectl -n test get triggers 
+    ``` 
+    **NOTE**: If any of the conditions above was not fulfilled, recreate the Application using the Kyma Console. If the           problem still exists, create a support ticket.
+
+5. If you added a service using an EventsServiceClass, check if the service instance is present in your Namespace:
+    
+    ```bash
+    kubectl -n test get serviceinstances.servicecatalog.k8s.io
```
Comment (k15r) [inline docs/migration-guides/1.10-1.11.md]:
```suggestion
    kubectl -n <YOUR_NAMESPACE> get serviceinstances.servicecatalog.k8s.io 
```

File: docs/migration-guides/1.10-1.11.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+5. If you added a service using an EventsServiceClass, check if the service instance is present in your Namespace:
+    
+    ```bash
+    kubectl -n test get serviceinstances.servicecatalog.k8s.io 
+    ```
+  
+6. Check if a Broker exisits in your Namespace:
+    
+    ```bash
+       kubectl -n test get brokers
```
Comment (k15r) [inline docs/migration-guides/1.10-1.11.md]:
```suggestion
       kubectl -n <YOUR_NAMESPACE> get brokers
```

also first check if the knative-eventing-injection is enabled for those namespaces

```bash
k get ns -lknative-eventing-injection=enabled
```

File: docs/migration-guides/1.10-1.11.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    
+7. Check if a Knative Subscription exists in the `kyma-integration` Namespace, linking the Knative Channel to the Knative        Broker: 
+
+    ```bash
+    kubectl -n kyma-integration get subscriptions.messaging.knative.dev 
+    ```
+8. Check if a subscriber URI in the Knative Subscription points to the Broker:
+
+    ```bash
+    kubectl -n kyma-integration get subscriptions.messaging.knative.dev brokersub-skqcq -o jsonpath='{ .spec.subscriber.uri }'
```
Comment (k15r) [inline docs/migration-guides/1.10-1.11.md]:
```suggestion
    kubectl -n kyma-integration get subscriptions.messaging.knative.dev -lapplication-name -o jsonpath='{ .spec.subscriber.uri }' 
```

File: docs/migration-guides/1.10-1.11.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    kubectl -n kyma-integration get subscriptions.messaging.knative.dev 
+    ```
+8. Check if a subscriber URI in the Knative Subscription points to the Broker:
+
+    ```bash
+    kubectl -n kyma-integration get subscriptions.messaging.knative.dev brokersub-skqcq -o jsonpath='{ .spec.subscriber.uri }' 
+    ```
+    **NOTE**: If any of the conditions above was not fulfilled, delete the service instance and recreate it using the Kyma  Console. If the problem persists, create a support ticket.
+    
+9. After the migration, you can still use the existing Event Bus. To check the compatibility, try sending events to the legacy endpoint at https://gateway.domain/commerce/v1/events.
```
Comment (k15r) [inline docs/migration-guides/1.10-1.11.md]: Correct me if im wrong:
After the migration this endpoint should be handled by the compatibility-layer which routes the events from the old endpoints to the sinks using the new mesh

File: docs/migration-guides/1.10-1.11.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
+# Migrate from v1.10 to v1.11
+
+## Knative Eventing Mesh 
+
+To use new [Knative Eventing Mesh](https://kyma-project.io/docs/master/components/knative-eventing-mesh/#overview-overview), you must upgrade Kyma from version 1.10 to version 1.11. To do so, follow the [Kyma upgrade guide](https://kyma-project.io/docs/#installation-upgrade-kyma).
```
[the remark this answers is not in the document]
Comment (montaro) [reply docs/migration-guides/1.10-1.11.md]: @bszwarc

File: docs/migration-guides/1.10-1.11.md (hunk 3 above)
Comment (montaro) [reply docs/migration-guides/1.10-1.11.md]: @k15r I would go for the same approach like following comment
`-n <YOUR_NAMESPACE>`

File: docs/migration-guides/1.10-1.11.md (hunk 7 above)
Comment (montaro) [reply docs/migration-guides/1.10-1.11.md]: Yes, it's now misleading after the rewriting. will fix it.

Comment (montaro) [reply docs/migration-guides/1.10-1.11.md]: @k15r This is the old version 
> Make sure the compatibility layer is working well

> Verify that sending events to the legacy endpoint https://gateway.domain/commerce >/v1/events using app connector still consumed by the subscribed lambda
> P.S. It’s advised to use the new Knative Eventing Mesh endpoint https://gateway.domain
> /commerce/events/events

File: docs/migration-guides/1.10-1.11.md (hunk 6 above)
Comment (montaro) [reply docs/migration-guides/1.10-1.11.md]: `-lapplication-name` is not valid.
the labels provided are:
```
    "labels": {
      "application-name": "test",
      "broker-namespace": "test"
    },
```
Where the applcation name actually is application name is actually `newapp` not `test`!
Any suggestions?

Comment (k15r) [reply docs/migration-guides/1.10-1.11.md]: ```kubectl -n kyma-integration get subscriptions.messaging.knative.dev -o jsonpath='{ .items[*].spec.subscriber.uri }' -lapplication-name```

File: docs/migration-guides/1.10-1.11.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+9. Check if a subscriber URI in the Knative Subscription points to the Broker:
+
+    ```bash
+    kubectl -n kyma-integration get subscriptions.messaging.knative.dev -o jsonpath='{ .items[*].spec.subscriber.uri }' -lapplication-name 
+    ```
+    **NOTE**: If any of the steps results in a failure, delete the service instance and recreate it using the Kyma  Console. If the problem persists, create a support ticket.
+    
+### Eventing endpoints
+
+After the migration, use the new Knative Eventing Mesh endpoint, `https://gateway.domain/APP_NAME/events/events`, to send events.
```
Comment (anishj0shi) [inline docs/migration-guides/1.10-1.11.md]: /events is repeated twice

**Pull request merged**